### PR TITLE
Set Cinder API and Backup replicas to 3 for HCI VA

### PR DESCRIPTION
**Pull request by fultonj**
Update HCI VA so that replicas for cinderAPI and cinderBackup are 3.

### CODE REVIEW

**Comment by fultonj** [comment]
What does the `kustomize` output look like?
```
[fultonj@runcible ~]$ cd architecture/examples/va/hci/
[fultonj@runcible hci{storage_replicas}]$ kustomize build | grep cinderAPI -A 25
      cinderAPI:
        override:
          service:
            internal:
              metadata:
                annotations:
                  metallb.universe.tf/address-pool: internalapi
                  metallb.universe.tf/allow-shared-ip: internalapi
                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
              spec:
                type: LoadBalancer
        replicas: 3
      cinderBackup:
        customServiceConfig: |
          [DEFAULT]
          backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
          backup_ceph_pool = backups
          backup_ceph_user = openstack
        networkAttachments:
        - storage
        replicas: 3
      cinderScheduler:
        replicas: 1
      cinderVolumes:
        ceph:
          customServiceConfig: |
[fultonj@runcible hci{storage_replicas}]$ 
```
What do those incremented values do with the the cinder operator?

```
[zuul@controller-0 ~]$ oc get pods | grep cinder
cinder-api-0                                                      2/2     Running     0               3d2h
cinder-api-1                                                      2/2     Running     0               2m8s
cinder-api-2                                                      2/2     Running     0               2m8s
cinder-backup-0                                                   2/2     Running     0               2m8s
cinder-backup-1                                                   2/2     Running     0               117s
cinder-backup-2                                                   2/2     Running     0               106s
cinder-db-purge-28599841-5x5fs                                    0/1     Completed   0               2d20h
cinder-db-purge-28601281-t4lkt                                    0/1     Completed   0               44h
cinder-db-purge-28602721-dgdjn                                    0/1     Completed   0               20h
cinder-scheduler-0                                                2/2     Running     0               3d2h
cinder-volume-ceph-0                                              2/2     Running     0               3d2h
[zuul@controller-0 ~]$ 
```

File: va/hci/edpm-post-ceph/nodeset/kustomization.yaml [hunk 1]
```diff
@@ -77,6 +77,28 @@ replacements:
           - spec.glance.template.glanceAPIs.default.replicas
         options:
           create: true
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.cinderAPI.replicas
```
Comment (ASBishop) [inline va/hci/edpm-post-ceph/nodeset/kustomization.yaml:83]: This all looks fine, but the rest of the cinder replacements are above L58. Here you've added these cinder replacements after glance stuff. Would it make more sense to group the cinder replacements together?

Comment (abays) [reply va/hci/edpm-post-ceph/nodeset/kustomization.yaml:83]: That would make sense to me too, but otherwise this looks good

Comment (fultonj) [reply va/hci/edpm-post-ceph/nodeset/kustomization.yaml]: OK, I've done that.

**Review by ASBishop — APPROVED** [review]
/lgtm

**Pull request merged**